src/components/viewSections/LicenseAmendments.js

```diff
@@ -75,9 +74,9 @@ export default class LicenseAmendments extends React.Component {
           contentData={license.amendments || []}
           formatter={{
             name: a => a.name,
-            status: a => a?.status?.label ?? <NoValue />,
-            startDate: a => (a.startDate ? <FormattedUTCDate value={a.startDate} /> : <NoValue />),
-            endDate: a => <LicenseEndDate license={a} />,
+            status: a => a?.status?.label,
```

Comment:
If @aditya-matukumalli is fine with us sometimes returning NULL and sometimes undefined then I'm good with these changes